src/tiled/scriptmodule.h

```diff
@@ -102,6 +108,11 @@ class ScriptModule : public QObject
     Q_INVOKABLE void registerTilesetFormat(const QString &shortName, QJSValue tilesetFormatObject);
     Q_INVOKABLE QJSValue registerTool(const QString &shortName, QJSValue toolObject);
 
+    Q_INVOKABLE Tiled::ScriptMapFormatWrapper *getMapFormat(const QString &shortName) const;
+    Q_INVOKABLE Tiled::ScriptMapFormatWrapper *getMapFormatByFile(const QString &fileName) const;
+    Q_INVOKABLE Tiled::ScriptTilesetFormatWrapper *getTilesetFormat(const QString &shortName) const;
+    Q_INVOKABLE Tiled::ScriptTilesetFormatWrapper *getTilesetFormatByFile(const QString &fileName) const;
```

Comment:
Naming style: Please remove the "get" prefix from all these methods. And I think `ForFile` would be more suitable than `ByFile`.

Reply:
Ok.

Reply:
Still learning the style.